src/BookNavigator/downloads/downloads.js

```diff
@@ -40,6 +40,16 @@ export class IABookDownloads extends LitElement {
     ));
   }
 
+  /**
+   * checks if downloads list contains an LCP option
+   * @return {boolean}
+   */
+  get hasLCPOption() {
+    const regex = /^(LCP)/g;
+    const lcpAvailable = this.downloads.some(option => option.type?.match(regex));
```

Comment:
So you are assuming that LCP will be available in both version always. Any change LCP available for PDF version only?

Reply:
> Any change LCP available for PDF or EPUB version only? 

unsure, but they can use any of the new option types to add a link so we should be covered. `option.type: 'LCP ePub' | 'LCP PDF'`